src/snowcli/cli/nativeapp/commands.py

```diff
@@ -71,3 +75,18 @@ def app_bundle(
     manager = NativeAppManager(project_path)
     manager.build_bundle()
     return MessageResult(f"Bundle generated at {manager.deploy_root}")
+
+
+@app.command("diff", hidden=True)
```

Comment:
From PR description:
> The use case is currently only within Native Apps, hence the command resides under native-apps module, and needs to stay hidden as it is only used internally, and we have not yet discussed exposing this command for all users.

Why can't we put it to `snow stage` and keep it hidden there?